deb/contents.go

```diff
@@ -19,7 +19,13 @@ type ContentsIndex struct {
 
 // NewContentsIndex creates empty ContentsIndex
 func NewContentsIndex(db database.Storage, repo PublishedRepo, component string, architecture string, udeb bool) *ContentsIndex {
-	return &ContentsIndex{db: db, repo: repo, component: component, architecture: architecture}
+	index := &ContentsIndex{db: db, repo: repo, component: component, architecture: architecture}
+
+	// clean up old values
+	key := index.Key("", "")
+	db.DeleteByPrefix(key)
```

Comment:
The key function of contents index does the following:
[]byte(fmt.Sprintf("xI%s_%s_%v$%s%s", refKey, index.architecture, index.udeb, path, pkg))

so in this case path and pkg are empty but the prefix remains as only "xI" (marking contents index entries) and refKey of the repository. So this should be safely only deleting entries of given contents index.